browser/profiles/brave_profile_impl.cc

```diff
@@ -60,6 +65,12 @@ BraveProfileImpl::BraveProfileImpl(
         base::BindOnce(&ProfileImpl::OnPrefsLoaded,
                        weak_ptr_factory_.GetWeakPtr(), create_mode, true));
   }
+
+#if defined(OS_ANDROID)
```

Comment:
this should be in brave_profile_prefs.cc `RegisterProfilePrefs` as a `SetDefaultPrefValue`

Reply:
Addressed in https://github.com/brave/brave-core/commit/5609091d35e01b846c2750846bf7fb64edcb992b